src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -994,7 +1190,9 @@ def _visit_reduce(self, node: itir.FunCall):
             )
 
             input_mapping = {
-                param: create_memlet_at(arg.value.data, nreduce_index)
+                param: dace.Memlet(data=arg.value.data, subset="0")
+                if arg.value.desc(self.context.body).shape == (1,)
+                else create_memlet_at(arg.value.data, nreduce_index)
```

Comment:
I think we should patch `Memlet` to allow pass either a string or a list of strings which is then interpreted as `', '.join()` because I have seed that a lot as well and it is really cumbersome to write.